data/Crazycoder122.json

```diff
@@ -0,0 +1,19 @@
+{
+   "name":"Krishnendu Dasgupta",
+   "displayStatsPublic":true,
+   "type":"personal",
+   "bio":"Programmer by passion and also a Musician. Trying to solve the toughest Algorithms of life through Music..",
+   "avatar":"https://github.com/Crazycoder122.png",
+   "links":[
+      {
+         "name":"Krishnendu Dasgupta",
+         "url":"https://twitter.com/KrishnenduDG",
+         "icon":"FaTwitter"
+      },
+      {
+         "name":"Krishnendu Dasgupta",
```

Comment:
What do you think on updating it to something like -  `LinkedIn`, or `Connect with me on LinkedIn`, etc?